client/webpack.config.node.js

```diff
@@ -127,10 +130,7 @@ const webpackConfig = {
 				include: shouldTranspileDependency,
 			} ),
 			fileLoader,
-			{
-				test: /\.(sc|sa|c)ss$/,
-				loader: 'ignore-loader',
-			},
+			scssConfig.loader,
```

Comment:
Could/should we still ignore everything except files suffixed with `.module.scss` ? 

>Why were we not doing [server-side CSS processing and extraction] before? Is there anything we should do differently?

I'm not sure exactly why we would have needed to do it previously. Am I correct in understanding that it's required now so that the server can generate the computed class names in the server-rendered markup? Since we would have previously had static class names, I would suppose that's the reason we wouldn't need to process the stylesheets. I'd imagine there's also a performance overhead in doing so, at least in contrast with ignoring them altogether.

Reply:
>Could/should we still ignore everything except files suffixed with .module.scss ?

Interesting. I think that would help with performance.

>Am I correct in understanding that it's required now so that the server can generate the computed class names in the server-rendered markup?

Yes, that is exactly why. The class names must match both in the client and the server-generated markup.